### PR TITLE
[release] Adding `7.28.1-rc.1` and `6.28.1-rc.1` to `releases.json`

### PR DESCRIPTION
**Pull request by sgnn7**
This adds the rc.1 definition into `releases.json` of 7.28.1 patch.

### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe how to test your changes

Write here in detail how you have tested your changes
and instructions on how this should be tested in QA.

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -22,6 +22,30 @@
         "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.28.1-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.1",
```
Comment (truthbk) [inline release.json:26]: Have we verified the tag is in place in the integrations-core repo?

File: release.json [hunk 2]
```diff
@@ -22,6 +22,30 @@
...
+        "JMXFETCH_HASH": "758717a2fd9c50247336318644e9c985d9a8a9845039d59a8793923b3f76f494",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.9",
+        "MACOS_BUILD_VERSION": "6.28.1-rc.1",
+        "WINDOWS_DDNPM_DRIVER": "release-signed",
+        "WINDOWS_DDNPM_VERSION": "0.98.2.git.68.90e7d1e",
+        "WINDOWS_DDNPM_SHASUM": "f6efeb772e011d7a5327b413cc60e5380c6d108ca61554803c8e6c3fadcf14c1"
+    },
+    "7.28.1-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.28.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.28.1-rc.1",
```
Comment (truthbk) [inline release.json:39]: were there changes to the omnibus-software and omnibus-ruby projects? If not, I don't think we need new tags there, it's prone to error.

**Review by truthbk — COMMENTED** [review]
Couple of nits, but looks (almost?) ready to go.

File: release.json (hunk 1 above)
Comment (sgnn7) [reply release.json:26]: Yeah: https://github.com/DataDog/integrations-core/tree/7.28.1

File: release.json (hunk 2 above)
Comment (sgnn7) [reply release.json:39]: - `omnibus-ruby` has the same commit for all tags (7.28.0, 7.28.1, and 7.28.1-rc.1)
- `omnibus-software` has 1 commit diff between 7.28.0 and others but it seems like a desired change: https://github.com/DataDog/omnibus-software/pull/396/files

**Review by truthbk — APPROVED** [review]

**Pull request merged**